constance/apps.py

```diff
@@ -8,4 +8,4 @@ class ConstanceConfig(AppConfig):
     default_auto_field = 'django.db.models.AutoField'
 
     def ready(self):
-        from . import checks
+        pass
```

Comment:
```suggestion
```

Reply:
To be honest, I believe we should keep that, it activates the checks.

Reply:
Checks are activated because of registeing via `register`

```
from django.core.checks import Error, register
@register()
....
```

Reply:
Yes but you need to call register and I believe the `ready()` call does load the code and trigger register.

Reply:
It's not so.
While you use register and place `"constance"` in `INSTALLED_APPS` you do not need to do more (like import)

Reply:
Ok thank you for checking :pray:

Reply:
I have tested with my project
```
❯ ./manage.py check --tag constance
System check identified no issues (0 silenced).
❯ ./manage.py check --tag not_constance
CommandError: There is no system check with the "not_constance" tag.
```

Reply:
https://github.com/jazzband/django-constance/pull/562/commits/d30759be5d94122054f95827e720d0654dcb3267

Reply:
I have checked this case attentively. I have just set breakpoint at the beginning of checks.py file.
If we remove import from apps.py then check no matter works.
But it works because of import in forms.py **from .checks import get_inconsistent_fieldnames**!
Thus correct work of checks is just a side effect of this import. If I move **import from .checks import get_inconsistent_fieldnames** from module import to  local in **def clean(self)** function then checks are not run.
Future refactoring can break checks in implicit manner.
I suggest to return this import to the place where it was.
Also it is recommended way by django documentation https://docs.djangoproject.com/en/5.0/topics/checks/#registering-and-labeling-checks

Reply:
Thank you. Fixed https://github.com/jazzband/django-constance/pull/562/commits/68a1c5036e87b9ce88f67b8cd6028197ddb85572